src/SFA.DAS.ApplyService.Database/Tables/AssessorPageReviewOutcome.sql

```diff
@@ -0,0 +1,25 @@
+﻿CREATE TABLE [dbo].[AssessorPageReviewOutcome](
+	[Id] [uniqueidentifier] NOT NULL,
+	[ApplicationId] [uniqueidentifier] NOT NULL,
+	[SequenceNumber] [int] NOT NULL,
+	[SectionNumber] [int] NOT NULL,
+	[PageId] [nvarchar](50) NOT NULL,
+	[Assessor1UserId] NVARCHAR(256) NULL,     
```

Comment:
Do we also need the assessor name?

Reply:
No. I don’t think that we need assessor name. We don’t need to display it anywhere.
We need IDs only to identify the records and update the correct ones.

Reply:
👍